app/resources/results.html.template

```diff
@@ -46,37 +46,33 @@
       // third party search API directly here, to workaround AJAX's same
       // origin policy, and for authorization.
       function perform_third_party_search(id) {
-        fetch(repo_url + '/third_party_search', {
-          method: 'POST',
-          body: JSON.stringify({
-            query: query,
-            lang: lang,
-            query_type: query_type,
-            search_engine_id: id,
-          }),
-          headers: {
-            'Content-Type': 'application/json',
-          },
-        })
-        .then(r => r.json())
-        .then(function(r) {
-          console_log('Third-party search response: ', r);
-          if (r.status == 'OK') {
-            if (typeof(r.url) != 'string') {
-              console_error(
+        var req = new XMLHttpRequest();
+        const url = (
+            repo_url + '/third_party_search?query=' + encodeURIComponent(query)
+            + '&lang=' + encodeURIComponent(lang) + '&query_type='
+            + encodeURIComponent(query_type) + '&search_engine_id='
+            + encodeURIComponent(id));
+        req.open('GET', url);
+        req.setRequestHeader(
```

Comment:
Since you switched back to a GET request, I guess you don't need Content-type here?

Reply:
Done.